pkg/virt-api/webhooks/validating-webhook/admitters/vmi-update-admitter_test.go

```diff
@@ -51,7 +51,9 @@ var _ = Describe("Validating VMIUpdate Admitter", func() {
 		},
 		Spec: v1.KubeVirtSpec{
 			Configuration: v1.KubeVirtConfiguration{
-				DeveloperConfiguration: &v1.DeveloperConfiguration{},
+				DeveloperConfiguration: &v1.DeveloperConfiguration{
```

Comment:
We should not enable the feature gate for all tests here. We should change
`config, _, _ := testutils.NewFakeClusterConfigUsingKV(kv)`
to 
`config, _, informer:= testutils.NewFakeClusterConfigUsingKV(kv)`

Then we can have the dedicated context to test `ExperimentalVirtiofsSupport` feature gate. In the context we can do
```
testutils.UpdateFakeKubeVirtClusterConfig(kvInformer, &v1.KubeVirt{
			Spec: v1.KubeVirtSpec{
				Configuration: v1.KubeVirtConfiguration{
					DeveloperConfiguration: &v1.DeveloperConfiguration{
                                            FeatureGates: []string{"ExperimentalVirtiofsSupport"},
                                        },
				},
			},
		})
```

Reply:
Is it acceptable to move the whole existing `DescribeTable("Should return proper admission response", ...)` into this new context or should I only move the `Entry()`s that actually *need* the feature gate into the dedicated context (using a second, similar `DescribeTable(...)`)?

Reply:
We should extract the function that is now inlined as an argument to `DescribeTable`:
```
DescribeTable("Should return proper admission response", func(newVolumes, oldVolumes []v1.Volume, newDisks, oldDisks []v1.Disk, filesystems []v1.Filesystem, volumeStatuses []v1.VolumeStatus, expected *admissionv1.AdmissionResponse) {
newVMI := api.NewMinimalVMI("testvmi")
```

to  
```
func testResponse(newVolumes, oldVolumes []v1.Volume, newDisks, oldDisks []v1.Disk, filesystems []v1.Filesystem, volumeStatuses []v1.VolumeStatus, expected *admissionv1.AdmissionResponse) {
```
and then we can modify the existing `DescribeTable` to

`DescribeTable("Should return proper admission response",  testResponse`

and create new `Context` where we can put new `It("something", testResponse)` or `DescribeTable`.